### PR TITLE
Safeguard Makefile commands

### PR DESCRIPTION
**Pull request by KoBeWi**
See https://chat.godotengine.org/channel/documentation?msg=ekvLRbP9Av5M2YuQj

I don't know how to test this though.

### CODE REVIEW

File: doc/Makefile [hunk 1]
```diff
@@ -1,28 +1,28 @@
 BASEDIR = .
-CLASSES = $(BASEDIR)/classes/ $(BASEDIR)/../modules/
+CLASSES = "$(BASEDIR)/classes/" "$(BASEDIR)/../modules/"
 OUTPUTDIR = $(BASEDIR)/_build
 TOOLSDIR = $(BASEDIR)/tools
```
Comment (akien-mga) [inline doc/Makefile:4]: Should those be quoted too for good measure?

**Comment by akien-mga** [comment]
To test you need GNU Make (possibly part of git-bash.exe, or in a MSYS2/mingw distribution) and run e.g. `make rst` in the `doc` folder. And e.g. `make rst BASEDIR=yolo/dontdeletemysystem`.

File: doc/Makefile (hunk 1 above)
Comment (KoBeWi) [reply doc/Makefile:4]: This would result in double quotes, e.g. `mkdir -p ""output"/rst"`

**Review by akien-mga — APPROVED** [review]

**Comment by KoBeWi** [comment]
Fixed. It's kind of inconsistent though.

**Comment by akien-mga** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.1.

**Pull request merged**